src/styles/blocks/card.scss

```diff
@@ -0,0 +1,117 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text-secondary-color;
```

Comment:
Ensure that SCSS variables like `$text-secondary-color` are placed in a separate variables file, as required by the task.